README.md

```diff
@@ -64,6 +65,15 @@ Additional build prerequisites are:
 - [uglifyjs](https://github.com/mishoo/UglifyJS) to minify Javascript builds.
 - [ccache](https://ccache.samba.org) (optional) recommended for much faster rebuilds.
 
+On Mac, you will also need:
+
+- [Homebrew](https://brew.sh/) for installing dependencies
+- System libraries in the root directory (`sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /` should do it, see https://github.com/pyenv/pyenv/issues/1219#issuecomment-428305417)
+- coreutils for md5sum and other essential Unix utilities (`brew install coreutils`)
+- openssl (`brew install openssl`)
+- gfortran (`brew cask install gfortran`)
+- f2c: Install wget (`brew install wget`), and then download and run this gist as root: https://gist.github.com/wlach/b5edcd895496298074e550952f0fe3ef
```

Comment:
Should we just include this gist in the `tools` directory here?

Reply:
Makes sense to me, will address before landing

Reply:
Would of course be nice to be able to skip this step, maybe something to address in a followup.